app/views/layouts/_flashes.html.erb

```diff
@@ -0,0 +1,8 @@
+<% flash.each do |key, value| %>
+  <% if key == "alert" %>
```

Comment:
why don't u use directly the "danger" key when you set up the flash. You can always do `flash[:my_custom_key]`